doc/structural.xxml

```diff
@@ -141,13 +141,14 @@
 <!-- doxrox-include igraph_similarity_inverse_log_weighted -->
 </section>
 
-<section id="trees"><title>Trees</title>
+<section id="trees"><title>Trees and Forest</title>
```

Comment:
"forests" should be plural.

"forests" should not be capitalized: let's keep the style consistent with the rest of this document.